Testing/Exec/CeptrTest/PelePhysicsSimulation/utils/plotFunctions.H

```diff
@@ -0,0 +1,29 @@
+void
```

Comment:
I think you can clean up the `utils` folder. 
I think you only need the `initFunctions.H` and the `reactFunctions.H` Also those are in the `IgnitionDelay` test case, so maybe we can remove some redundancy here?